logging/zap_logger.go

```diff
@@ -61,26 +61,6 @@ func (z *ZapLogger) Fatal(msg string, tags ...any) {
 	z.logger.Sugar().Fatalw(msg, tags...)
 }
 
-func (z *ZapLogger) Debugm(msg string, tags map[string]any) {
-	z.logger.Debug(msg, convertTagsToZapFields(tags)...)
-}
-
-func (z *ZapLogger) Infom(msg string, tags map[string]any) {
-	z.logger.Info(msg, convertTagsToZapFields(tags)...)
-}
-
-func (z *ZapLogger) Warnm(msg string, tags map[string]any) {
-	z.logger.Warn(msg, convertTagsToZapFields(tags)...)
-}
-
-func (z *ZapLogger) Errorm(msg string, tags map[string]any) {
-	z.logger.Error(msg, convertTagsToZapFields(tags)...)
-}
-
-func (z *ZapLogger) Fatalm(msg string, tags map[string]any) {
```

Comment:
you had previously removed these from the interface, but forgotten to delete them on the struct here, so cleaning up? Is this what's happening?

Reply:
yea! we don't use tags map anymore.